src/components/ExpandCollapse/ExpandCollapse.tsx

```diff
@@ -7,12 +7,15 @@ import { VibeComponentProps } from "../../types";
 import cx from "classnames";
 import { ElementContent } from "../../types/ElementContent";
 import "./ExpandCollapse.scss";
+import extraStyles from "./styles.module.scss";
```

Comment:
Maybe to rename the file to`ExpandCollapse.module.scss` or smth? Just to keep the convention

Reply:
Or maybe it's better to leave it this way - would be less confusing when I'll merge with Vibe 2.0

Reply:
thought about 2.0 so afterwards we would just merge the both